Only in .: patch-x11.patch
diff -u ./qxtglobalshortcut_x11.cpp ../qxt-1/qxtglobalshortcut_x11.cpp
--- ./qxtglobalshortcut_x11.cpp	2016-12-07 00:54:54.000000000 +0300
+++ ../qxt-1/qxtglobalshortcut_x11.cpp	2016-12-07 00:54:43.000000000 +0300
@@ -30,7 +30,6 @@
 *****************************************************************************/
 
 #include <QVector>
-#include <X11/Xlib.h>
 #if QT_VERSION < QT_VERSION_CHECK(5,0,0)
 #   include <QX11Info>
 #else
@@ -38,6 +37,7 @@
 #   include <qpa/qplatformnativeinterface.h>
 #   include <xcb/xcb.h>
 #endif
+#include <X11/Xlib.h>
 
 namespace {
 
